Source/Core/VideoCommon/VertexLoaderManager.h

```diff
@@ -42,8 +42,11 @@ NativeVertexFormat* GetOrCreateMatchingFormat(const PortableVertexDeclaration& d
 NativeVertexFormat* GetUberVertexFormat(const PortableVertexDeclaration& decl);
 
 // Returns -1 if buf_size is insufficient, else the amount of bytes consumed
-int RunVertices(int vtx_attr_group, OpcodeDecoder::Primitive primitive, int count, DataReader src,
-                bool is_preprocess);
+template <bool IsPreprocess = false>
+int RunVertices(int vtx_attr_group, OpcodeDecoder::Primitive primitive, int count, DataReader src);
+
+template <bool IsPreprocess = false>
+VertexLoaderBase* GetOrCreateLoader(int vtx_attr_group);
```

Comment:
I think this needs a comment mentioning that `RefreshLoader` is the recommended function as it checks `attr_dirty`. If it were easy, I'd recommend making this function private/friend to only `RefreshLoader` but it's not currently in a class so that's not possible.  Perhaps putting it in a `namespace detail` (within `namespace VertexLoaderManager`) would work?

Reply:
Done.